v2/group/events/member_updated.avsc

```diff
@@ -0,0 +1,20 @@
+{
+  "type":"record",
+  "name":"MemberGroupUpdated",
```

Comment:
```suggestion
  "name":"GroupMemberUpdated",
```

This is incredibly picky. So ignore this and the other 2 similar suggestions if you want. But since we are doing a v2 I think this is easier to say and it matches the naming logic for project.